tests/test_checks.py

```diff
@@ -0,0 +1,142 @@
+
+    @override_settings(CORS_ALLOW_METHODS=1)
+    def test_cors_allow_methods_failure(self):
+        errors = check_settings(corsheaders_settings)
+        assert len(errors) > 0
+        assert isinstance(errors[0], checks.Error)
+        assert errors[0].id == "corsheaders.W002"
+
+    @override_settings(CORS_ALLOW_CREDENTIALS=())
+    def test_cors_allow_credentials_failure(self):
+        errors = check_settings(corsheaders_settings)
+        assert len(errors) > 0
+        assert isinstance(errors[0], checks.Error)
+        assert errors[0].id == "corsheaders.W003"
+
+    @override_settings(CORS_PREFLIGHT_MAX_AGE="10")
+    def test_cors_preflight_max_age_failure(self):
+        errors = check_settings(corsheaders_settings)
+        assert len(errors) > 0
+        assert isinstance(errors[0], checks.Error)
+        assert errors[0].id == "corsheaders.W004"
+
+    @override_settings(CORS_ORIGIN_ALLOW_ALL=0)
+    def test_cors_origin_allow_all_failure(self):
+        errors = check_settings(corsheaders_settings)
+        assert len(errors) > 0
+        assert isinstance(errors[0], checks.Error)
+        assert errors[0].id == "corsheaders.W005"
+
+    @override_settings(CORS_ORIGIN_WHITELIST=True)
+    def test_cors_origin_whitelist_failure(self):
+        errors = check_settings(corsheaders_settings)
+        assert len(errors) > 0
+        assert isinstance(errors[0], checks.Error)
+        assert errors[0].id == "corsheaders.W006"
+
+    @override_settings(CORS_ORIGIN_REGEX_WHITELIST=10)
+    def test_cors_origin_regex_whitelist_failure(self):
+        errors = check_settings(corsheaders_settings)
+        assert len(errors) > 0
+        assert isinstance(errors[0], checks.Error)
+        assert errors[0].id == "corsheaders.W007"
+
+    @override_settings(CORS_EXPOSE_HEADERS=True)
+    def test_cors_expose_headers_failure(self):
+        errors = check_settings(corsheaders_settings)
+        assert len(errors) > 0
+        assert isinstance(errors[0], checks.Error)
+        assert errors[0].id == "corsheaders.W008"
+
+    @override_settings(CORS_URLS_REGEX=())
+    def test_cors_urls_regex_failure(self):
+        errors = check_settings(corsheaders_settings)
+        assert len(errors) > 0
+        assert isinstance(errors[0], checks.Error)
+        assert errors[0].id == "corsheaders.W009"
+
+    @override_settings(CORS_MODEL=())
+    def test_cors_model_failure(self):
+        errors = check_settings(corsheaders_settings)
+        assert len(errors) > 0
+        assert isinstance(errors[0], checks.Error)
+        assert errors[0].id == "corsheaders.W0010"
+
+    @override_settings(CORS_REPLACE_HTTPS_REFERER=())
+    def test_cors_replace_https_referer_failure(self):
+        errors = check_settings(corsheaders_settings)
+        assert len(errors) > 0
+        assert isinstance(errors[0], checks.Error)
+        assert errors[0].id == "corsheaders.W0011"
+
+    @override_settings(CORS_ALLOW_HEADERS=default_headers)
```

Comment:
All the below tests are just setting the settings to default, which they're all already set to. You should combine them into a single test `test_defaults_pass`, with no `override_settings`